cycode/cli/printers/utils/detection_ordering/common_ordering.py

```diff
@@ -37,22 +36,14 @@ def _sort_detections_by_file_path(
 def sort_and_group_detections(
     detections_with_documents: List[Tuple['Detection', 'Document']],
 ) -> GroupedDetections:
-    """Sort detections by severity and group by file name."""
-    detections = []
+    """Sort detections by severity. We do not have groping here (don't find the best one yet)."""
```

Comment:
```suggestion
    """Sort detections by severity. We do not have grouping here (don't find the best one yet)."""
```

Reply:
thank you! i will apply it in the next PR; dont want to bother CI once again